files/rsyslog/55-elasticsearch.conf

```diff
@@ -61,6 +61,8 @@ ruleset(name="try_es") {
             queue.maxFileSize=`echo $ES_QUEUE_MAXFILESIZE`
             queue.checkpointInterval=`echo $ES_QUEUE_CHECKPOINTINTERVAL`
             queue.syncqueuefiles="on"
+            action.resumeRetryCount="-1"
+            action.resumeIntervalMax=`echo $ES_RETRY_WAIT`
```

Comment:
I cannot find resumeIntervalMax but resumeInterval in the doc and the sources...  Could you point me to the reference?

Reply:
good catch - `action.resumeIntervalMax` is not supported in our version of rsyslog, even though it is listed on the page, and doesn't say `new in 8.xx version` - https://www.rsyslog.com/doc/v8-stable/configuration/actions.html#general-action-parameters